### PR TITLE
Feature/us4266 group details participants view and remove

### PR DESCRIPTION
**Pull request by IngageKrizJ**
* [US4266](https://rally1.rallydev.com/#/27593764268d/detail/userstory/58429571573)

### CODE REVIEW

File: Gateway/crds-angular/Controllers/API/GroupToolController.cs [hunk 1]
```diff
@@ -77,5 +78,30 @@ public IHttpActionResult GetInquiries(int groupId)
                 }
             });
         }
+
+        [RequiresAuthorization]
+        [Route("api/grouptool/grouptype/{groupTypeId:int}/group/{groupId:int}/participant/{groupParticipantId:int}")]
```
Comment (dtkocher) [inline Gateway/crds-angular/Controllers/API/GroupToolController.cs]: @IngageKrizJ Is the :int stuff new to VS 2015? haven't seen it before

Comment (IngageKrizJ) [inline Gateway/crds-angular/Controllers/API/GroupToolController.cs]: No, it's part of WebAPI, we used it for a bunch of the giving controllers too.  You can do all sorts of validations on the URI.

File: crossroads.net/app/group_tool/group_detail/participants/groupDetail.participants.controller.js [hunk 2]
```diff
@@ -1,22 +1,43 @@
...
+
+    this.setListView();
   }
 
   $onInit() {
+    this.participantService.get().then((myParticipant) => {
+      this.myParticipantId = myParticipant.ParticipantId;
+      this.loadGroupParticipants();
+    }, (err) => {
+      this.log.error(`Unable to get my participant: ${err.status} - ${err.statusText}`);
```
Comment (dtkocher) [inline crossroads.net/app/group_tool/group_detail/participants/groupDetail.participants.controller.js]: @IngageKrizJ Should we emit an error here?

**Comment by jpc11623** [comment]
Reviewed, looks good.

**Pull request merged**